src/fusedav-statsd.c

```diff
@@ -207,6 +207,20 @@ int stats_close(void) {
     return 0;
 }
 
+/*  Does it pass the sampling rate? */
+static int sample(float samplerate) {
+    const char *funcname = "sample";
+    if (samplerate < 1.0) {
+        long int value = random();
+        float sample = ((float)value / RAND_MAX);
+        log_print(LOG_DEBUG, SECTION_STATS_DEFAULT, "%s: sample: %.2f, rate: %.2f, send: %d", 
+                funcname, sample, samplerate, (samplerate > sample));
+        return samplerate > sample;
+    } else {
```

Comment:
just a suggestion here:  the else is unnecessary here.  But if the fusedav codebase currently does this in most places then it probably makes sense to stay consistent

from mozilla coding style guide:
> Don't put an else right after a return (or a break). Delete the else, it's unnecessary and increases indentation level.

Reply:
OMG. I have scars from all those wars!